control-plane/agents/src/bin/core/tests/event/mod.rs

```diff
@@ -0,0 +1,217 @@
+use deployer_cluster::{Cluster, ClusterBuilder};
+use events_api::{
+    event::{EventAction, EventCategory, EventMessage},
+    mbus_nats::{message_bus_init, BusSubscription},
+    Bus,
+};
+use grpc::operations::{
+    nexus::traits::NexusOperations, pool::traits::PoolOperations, volume::traits::VolumeOperations,
+};
+use std::{collections::HashMap, time::Duration};
+use stor_port::types::v0::{
+    openapi::{
+        models,
+        models::{CreateVolumeBody, PublishVolumeBody, VolumePolicy},
+    },
+    transport::{DestroyNexus, DestroyPool, DestroyVolume, NexusId, UnpublishVolume, VolumeId},
+};
+use tokio::time::timeout;
+
+const CHILD_WAIT: u64 = 5;
```

Comment:
Don't make reader guess units by using Duration.

```suggestion
const CHILD_WAIT: Duration = Duration::from_secs(5);
```